modules/test/playwright/pages/account-admin-web/AccountEntriesManagementPortletPage.ts

```diff
@@ -0,0 +1,148 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {Locator, Page} from '@playwright/test';
+
+import {CommerceLayoutsPage} from '../commerce/commerceLayoutsPage';
+
+export const searchTableRowByValue = async function (
+	tableLocator: Locator,
+	colPosition: number,
+	value: string,
+	strictEqual: boolean = false
+) {
+	await tableLocator.elementHandle();
+
+	const rows = await tableLocator.getByRole('row').all();
+
+	for await (const row of rows) {
+		const column = row.getByRole('cell').nth(colPosition).first();
+
+		const colValue = (await column.allInnerTexts()).join('');
+
+		if (
+			(strictEqual && colValue === value) ||
+			(!strictEqual &&
+				colValue.toLowerCase().indexOf(value.toLowerCase()) >= 0)
+		) {
+			return {column, row};
+		}
+	}
+
+	throw new Error(`Cannot locate table row with value ${value}`);
+};
+
+export class AccountEntriesManagementPortletPage {
```

Comment:
DNDTable doesn't work here since this page uses a searchContainer, but i imported the function from UsersAndOrganizationsPage instead of copying it here. I think in the future we could also create something like CommerceSearchContainerTablePage, since this function is being imported and used in various other pages as well.